### PR TITLE
Add Unfollow Playlist for Current User

### PR DESCRIPTION
**Pull request by armineprox**
Added Operation for https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-playlist/

### CODE REVIEW

**Comment by dargmuesli** [comment]
Thanks for contributing, I'll take a closer look at it soon :)

**Comment by dargmuesli** [comment]
Do you plan to create the same PR for https://developer.spotify.com/documentation/web-api/reference/follow/follow-playlist/?

**Comment by armineprox** [comment]
Yes! And now i can follow the`legacy` directory naming pattern :) Can we merge this one first, though? I'd like to use it for a side project of mine :)

**Comment by dargmuesli** [comment]
I'd prefer to merge both at once.

File: src/main/java/com/wrapper/spotify/requests/data/follow/legacy/UnfollowPlaylistRequest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package com.wrapper.spotify.requests.data.follow;
```
Comment (armineprox) [inline src/main/java/com/wrapper/spotify/requests/data/follow/legacy/UnfollowPlaylistRequest.java]: @dargmuesli Wrong package here. Changing it now.

**Comment by armineprox** [comment]
@dargmuesli what is the release/publishing process for this project to maven/sbt?

**Comment by dargmuesli** [comment]
Travis releases the update automatically for tags. But Travis somehow doesn't build this project currently. I have no idea why and am investigating for quite some time now.

**Comment by armineprox** [comment]
Gotcha! Thank you so much! Fyi, Github was having issues delivering webhooks. Might be related to that.

**Comment by dargmuesli** [comment]
Thanks for the information, maybe I've been searching at the wrong end then (Travis). As I'm only the maintainer I cannot check the webhook setup for this project. I'll try my best, sorry for the delay :)

**Comment by dargmuesli** [comment]
The Travis build issue was fixed by the Travis support, but we have another issue with gpg at the moment.

**Comment by dargmuesli** [comment]
Version 2.2.0 is now available from the repositories.

**Pull request merged**